packages/flutter_tools/templates/plugin_example/ios-swift.tmpl/RunnerTests/pluginClassTests.swift.tmpl

```diff
@@ -0,0 +1,31 @@
+import Flutter
+import UIKit
+import XCTest
+
+@testable import {{pluginProjectName}}
+
+// This demonstrates a simple unit test of the Swift portion of this plugin's implementation.
+//
+// IMPORTANT: Before these tests can be compiled and run, the following needs to be added to this
+// example application's Podfile, in the `target 'Runner' do` section:
+//     target 'RunnerTests' do
+//       inherit! :search_paths
+//     end
+// followed by running 'pod install' in the example/ios/ directory.
```

Comment:
This makes me sad, but I couldn't think of solution that doesn't involve running `pod install` during template instantiation which seemed... not great. Even with this limitation I think this is a huge improvement since "edit one text file" is a much easier thing for people with little-to-no Xcode experience than "make a new target and configure it correctly, then set up the test source structure including correct imports and test flow". But If you have ideas on how to make this unnecessary that would be great.

Reply:
This problem would go away if we always add the `RunnerTest` since it would always be in the Podfile.

But even if that isn't done, can't we conditionalize in the Podfile in the same way?

Reply:
I forgot that we were creating it from a different template stage 🤦🏻  I'll take a look at parameterization options at that stage.